llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -120,7 +120,9 @@ InstructionCost RISCVTTIImpl::getIntImmCost(const APInt &Imm, Type *Ty,
 
   // Otherwise, we check how many instructions it will take to materialise.
   const DataLayout &DL = getDataLayout();
-  return RISCVMatInt::getIntMatCost(Imm, DL.getTypeSizeInBits(Ty), *getST());
+  return RISCVMatInt::getIntMatCost(Imm, DL.getTypeSizeInBits(Ty), *getST(),
```

Comment:
Just noting that the special case at 118 is no longer required.  Fine to leave for perf reasons, but no longer semantically relevant.